### ISSUE TITLE
multiple class attributes per HTML tag are not supported

### ISSUE THREAD

**Issue by Parcley** [neuland/jade4j#129]
Having a div-element with multiple classes attributes in a jade file will produce a HTML output with only one class attribute (the last declared attribute will overwrite all attributes declared before)
multiple classes are supported in the original Jade, see http://jade-lang.com/reference/attributes/

trivial example:

by using this jade template
```jade
doctype html
html
  body
    div(class='foo' class='bar')
```

the HTML output I get from jade4j is
```html
<!DOCTYPE html>
<html>
  <body>
    <div class="bar"></div>
  </body>
</html>
```

the expected HTML output would be 
```html
<!DOCTYPE html>
<html>
  <body>
    <div class="foo bar"></div>
  </body>
</html>
```

**Comment by lcsjunior** [comment]
div.foo.bar ??

**Comment by chbloemer** [comment]
div.foo.bar works.
div(class='foo' class='bar') doesn't work.

**Comment by chbloemer** [comment]
Added testcase issues/129.jade. Fixed in Master Branch.

**Comment by chbloemer** [comment]
Fixed in release 1.2.0
